engines/bastion_katello/app/assets/javascripts/bastion_katello/products/details/repositories/details/views/repository-info.html

```diff
@@ -82,7 +82,7 @@ <h4 translate>Sync Settings</h4>
           readonly="denied('edit_products', product)">
       </dd>
 
-      <span ng-show="!product.redhat">
+      <span ng-if="!product.redhat">
```

Comment:
I was able to reproduce after performing a pirouette on my head while juggling several bowling pins with my feet - and it fixes the problem! :+1: 

I noticed one thing that seems to related to s/ng-show/ng-if -> when I load the repo details page for a red hat product the 'Upstream Authorization' fields is shown and disappears shortly after (removal from DOM). Though I think it's the crux of your change - is it required / can be done in a different way?

Reply:
I could change this to ng-if="product && !product.redhat"  I think that would make it 'pop in' to screen rather than 'pop out'.  I would think this would happen even with ng-show though, but i'm not quite sure.

Reply:
Also its probably less of an issue with a production server where multiple requests can be handled at the same time (and faster)

Reply:
A similar behavior happens on master - I guess I'm just not looking at repo details for Red Hat products too much. You're right - should be better in prod, so I'm OK with it.